modules/mod_ginger_base/mod_ginger_base.erl

```diff
@@ -168,9 +168,7 @@ manage_schema(_Version, Context) ->
                 ]}
             ]}
         ]
-    },
-    z_datamodel:manage(?MODULE, Datamodel, Context),
-    schema:create_identity_if_not_exists(editor_dev, "redacteur", "redacteur", Context).
```

Comment:
Where did this go?

Reply:
It went nowhere, as I think it is not used / we should not use it (here) and rethink it when we need it.

Reply:
Use `manage_data/2`, it is called after `manage_schema/2`

```erlang
manage_data(_Version, Context) ->
    %% Whatever data needs to be installed after the datamodel
    %% has been installed.
    ok.
```